src/test/regress/sql/global_cancel.sql

```diff
@@ -48,4 +48,6 @@ SELECT pg_terminate_backend(10000000000 * :coordinator_node_id + 0);
 
 RESET client_min_messages;
 
+SELECT citus_backend_gpid() = citus_calculate_gpid(:coordinator_node_id, pg_backend_pid());
```

Comment:
Lovely test ❤️

Reply:
look at the other test, it is even nicer:p